content/en/project/getInvolved/_index.md

```diff
@@ -50,7 +50,7 @@ The archives are public. The LispCore group gets much more email, and posts from
 * X (Twitter) handle: [interlisp8](https://twitter.com/interlisp8).
 * _Direct email:_ In addition to the email groups listed above, you can email us at [info@interlisp.org](mailto:info@interlisp.org).
 
-## 4. Weekly meetings. We have weekly meetings on Zoom:
+## 4. Weekly meetings. We have weekly video conferencing meetings:
 
 Mondays:    2pm PT/5pm ET. Implementation -- technical discussions  
 Wednesdays: 10am PT/1pm ET. Communication -- website, talks,
```

Comment:
Should we also update the meeting times?  10:30 am PT/1:30 pm ET.

Reply:
I updated the meeting times.